### PR TITLE
TPM event log fixups

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/hughsie/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c [hunk 1]
```diff
@@ -115,19 +115,25 @@ fu_plugin_device_registered_uefi (FuPlugin *plugin, FuDevice *device)
...
 			if (g_strcmp0 (checksum, checksum_tmp) == 0) {
 				data->reconstructed = TRUE;
-				return;
+				break;
 			}
 		}
+		/* check at least one reconstruction for this algorithm */
+		if (!data->reconstructed) {
+			fu_device_set_update_message (device,
+						     "TPM PCR0 %s differs from reconstruction, "
```
Comment (hughsie) [inline plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c:134]: I think you need to `g_strdup_printf` if you're using %s here.

File: plugins/tpm-eventlog/fu-tpm-eventlog.c [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #include "config.h"
 
 #include <fwupd.h>
+#include "fwupd-common-private.h"
```
Comment (hughsie) [inline plugins/tpm-eventlog/fu-tpm-eventlog.c:12]: maybe move this down a bit after the <> defines?

File: plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c (hunk 1 above)
Comment (superm1) [reply plugins/tpm-eventlog/fu-plugin-tpm-eventlog.c:134]: I think that %s was a typo when I moved it.  will fix

**Comment by hughsie** [comment]
LGTM, modulo a few niggles.

**Pull request merged**